uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/GoogleSettings.java

```diff
@@ -307,7 +307,7 @@ private void handleDoneButton() throws UiObjectNotFoundException {
         if (android.os.Build.VERSION.SDK_INT == 28) {
             UiAutomatorUtils.handleButtonClick("com.android.settings:id/redaction_done_button");
         } else {
-            final UiObject doneButton = UiAutomatorUtils.obtainUiObjectWithExactText("Done");
+            final UiObject doneButton = UiAutomatorUtils.obtainUiObjectWithExactText("DONE");
```

Comment:
Which device are you testing on? I don't expect this to change for API 29

Reply:
Pixel 4 emulator

Reply:
I see that this is still Done in the physical device. reverting this. For emulator on Pixel 4, i have to make this change locally.